src/execution/ast/ast_pretty_print.cpp

```diff
@@ -0,0 +1,360 @@
+#include "execution/ast/ast_pretty_print.h"
+
+#include <iostream>
+
+#include "execution/ast/ast_visitor.h"
+
+namespace terrier::execution::ast {
+
+std::string CastKindToInt(CastKind x) {
+  switch (x) {
+    // Conversion of a 32-bit integer into a non-nullable SQL Integer value
+    case CastKind::IntToSqlInt:
+      return "IntToSqlInt";
+
+      // Conversion of a 32-bit integer into a non-nullable SQL Decimal value
+    case CastKind::IntToSqlDecimal:
+      return "IntToSqlDecimal";
+
+      // Conversion of a SQL boolean value (potentially nullable) into a primitive
+      // boolean value
+    case CastKind::SqlBoolToBool:
+      return "SqlBoolToBool";
+
+      // Conversion of a primitive boolean into a SQL boolean
+    case CastKind::BoolToSqlBool:
+      return "BoolToSqlBool";
+
+      // A cast between integral types (i.e., 8-bit, 16-bit, 32-bit, or 64-bit
+      // numbers), excluding to boolean! Boils down to a bitcast, a truncation:
+      // a sign-extension, or a zero-extension. The same as in C/C++.
+    case CastKind::IntegralCast:
+      return "IntegralCast";
+
+      // An integer to float cast. Only allows widening.
+    case CastKind::IntToFloat:
+      return "IntToFloat";
+
+      // A float to integer cast. Only allows widening.
+    case CastKind::FloatToInt:
+      return "FloatToInt";
+
+      // A simple bit cast reinterpretation
+    case CastKind::BitCast:
+      return "BitCast";
+
+      // 64 bit float To Sql Real
+    case CastKind::FloatToSqlReal:
+      return "FloatToSqlReal";
+
+      // Conversion of a SQL timestamp value (potentially nullable) into a primitive timestamp value
+    case CastKind::SqlTimestampToTimestamp:
+      return "SqlTimestampToTimestamp";
+
+      // Conversion of a primitive timestamp valueinto a SQL timestamp
+    case CastKind::TimestampToSqlTimestamp:
+      return "TimestampToSqlTimestamp";
+
+      // Convert a SQL integer into a SQL real
+    case CastKind::SqlIntToSqlReal:
+      return "SqlIntToSqlReal";
```

Comment:
I think the comments here should have the same indent.

Reply:
Fixed in latest commit